packages/api-request-builder/src/delete.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+/**
+ * Set the `version` number to the internal state of the service instance
+ * in order to generate a delete uri
+ *
+ * @param  {number} version - The version of the item
+ * @throws if `version` is missing or not a number.
+ * @return {Object} The instance of the service, can be chained.
+ */
+
+export default function toDelete (version: number): Object {
+  if (!version || typeof version !== 'number')
+    throw new Error('Required version to delete missing or not valid')
```

Comment:
Yep, but I would expect there a word "IS" like this: `Required version to delete IS missing or it is not valid`